sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOIT.java

```diff
@@ -114,6 +130,17 @@ public class KafkaIOIT {
 
   @Rule public TestPipeline readPipeline = TestPipeline.create();
 
+  private static ExperimentalOptions pipelineOptions;
```

Comment:
consider naming `sdfPipelineOptions` since it is used only by `sdfReadPipeline` and put the initialization in setUp